ui/StatusQ/src/StatusQ/Layout/StatusAppNavBar.qml

```diff
@@ -1,195 +1,130 @@
-                anchors.top: parent.top
-                visible: parent.necessaryHightForCommunities > parent.maxHightForCommunities
-
-                Rectangle {
-                    height: 1
-                    width: 30
-                    color: Theme.palette.directColor7
-                    anchors.top: parent.top
-                    anchors.horizontalCenter: parent.horizontalCenter
-                }
-            }
-
-            ListView {
-                id: communityListView
-                anchors.left: parent.left
-                anchors.right: parent.right
-                anchors.top: separatorTop.visible? separatorTop.bottom : parent.top
-                anchors.bottom: separatorBottom.top
-                clip: true
-
-                boundsBehavior: contentHeight > height ? Flickable.DragAndOvershootBounds : Flickable.StopAtBounds
-                spacing: navBarButtonSpacing
-
-                model: navBarCommunityModel
-            }
-
-            Item {
-                id: separatorBottom
-                width: parent.width
-                height: navBarButtonSpacing
-                anchors.bottom: parent.bottom
-
-                Rectangle {
-                    height: 1
-                    width: 30
-                    color: Theme.palette.directColor7
-                    anchors.bottom: parent.bottom
-                    anchors.horizontalCenter: parent.horizontalCenter
-                }
-            }
+            objectName: "statusMainNavBarListView"
+
+            clip: true
+            spacing: d.spacing
+            boundsBehavior: contentHeight > height ? Flickable.DragAndOvershootBounds : Flickable.StopAtBounds
         }
-    }
 
-    ListView {
-        id: navBarListView
-        objectName: "statusMainNavBarListView"
-        anchors.left: parent.left
-        anchors.right: parent.right
-        anchors.top: parent.top
-        anchors.topMargin: 48
-        anchors.bottom: navBarProfileButtonSlot.top
-        anchors.bottomMargin: navBarButtonSpacing
+        Item {
+            Layout.fillWidth: true
+            Layout.fillHeight: true
+        }
 
-        spacing: navBarButtonSpacing
+        Loader {
+            id: cameraItemLoader
+            Layout.alignment: Qt.AlignHCenter
+        }
 
-        boundsBehavior: Flickable.StopAtBounds
-        model: navBarModel
+        Loader {
+            id: profileItemLoader
+            Layout.alignment: Qt.AlignHCenter
+        }
     }
 
-    Item {
-        id: navBarCameraButtonSlot
-        anchors.horizontalCenter: parent.horizontalCenter
-        height: visible? statusAppNavBar.navBarProfileButton.height : 0
-        width: visible? statusAppNavBar.navBarProfileButton.width : 0
-        visible: !!statusAppNavBar.navBarCameraButton
-        anchors.bottom: navBarProfileButtonSlot.visible ? navBarProfileButtonSlot.top : parent.bottom
-        anchors.bottomMargin: visible ? 12 : 0
-    }
+    QtObject {
+        id: d
```

Comment:
I think we decided that the `d` Object should be put at the top of the root, but I could be mistaken. Not gonna ask for changes, just asking/pointing it out.